apps/stats-dapp/src/pages/TransactionProgressCardShowCase.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import EthLogo from '@webb-tools/logos/chains/EthLogo';
+import {
+  TransactionPayload,
+  TransactionQueueCard,
+} from '@webb-tools/webb-ui-components/containers/TransactionProgressCard';
```

Comment:
We no longer need to make use of showcase, and should remove all showcase related components. Please remove this file and if you could remove the other unnecessary showcase components as well. Instead make use of storybook stories to display new component as you did.

Reply:
Sure that makes sense, removing it with he others